java/configuration/src/test/java/sleeper/configuration/properties/SleeperPropertiesTest.java

```diff
@@ -160,6 +161,19 @@ void shouldReturnNullIfListIsNullOrUnset() {
         assertThat(list).isNull();
     }
 
+    @Test
+    void shouldReturnEmptyListIfListPropertyIsExplicitlyUnset() throws IOException {
+        // Given
+        TestSleeperProperties testSleeperProperties = new TestSleeperProperties();
+        testSleeperProperties.loadFromString("sleeper.optional.stacks=");
```

Comment:
Would this be better as `(OPTIONAL_STACKS + "=");`?

Reply:
I'm not sure, this way it shows what the properties file will look like a bit clearer. I could change it to 
```java
testSleeperProperties.set(OPTIONAL_STACKS, "")
```
and update the test name to reflect this?

Reply:
OK, happy to leave it as it is.